src/schema/org.ts

```diff
@@ -4,9 +4,25 @@ import { Org as DefinedOrg } from '@/database/types'
 
 export const orgSchema = z.object({
     slug: z.string().regex(/^[a-z0-9-]+$/, { message: 'Invalid slug: must be all lowercase alphanumeric or dashes' }),
-    name: z.string().min(1, { message: 'Name must be provided' }),
+    name: z
+        .string()
+        .trim()
+        .min(1, 'Name is required')
+        .max(50, 'Name cannot exceed 50 characters')
+        .refine((val) => /\p{L}/u.test(val), {
+            message: 'Name must contain at least one alphabetic character',
+        })
+        .refine((val) => !/^\d+$/.test(val) || /\p{L}/u.test(val), {
```

Comment:
these could be combined into a single `refine`:
```suggestion
        .refine((val) => {
           if (/\p{L}/u.test(val)) return { message: 'Name must contain at least one alphabetic character' }
           if (!/^\d+$/.test(val)) return { message: 'Name cannot be only numbers' }
           }
```